schema/deploy/tables/report_type.sql

```diff
@@ -5,7 +5,9 @@ begin;
 
 create table cif.report_type
 (
-  name varchar(1000) primary key
+  name varchar(1000) primary key,
+  form_schema jsonb,
```

Comment:
are we planning to use this? If we were to store form schemas in the db, I'd have them in a separate table instead

Reply:
yeah, this should be a foreign key that's added whenever we implement versioned form schemas. I'll remove it here and take it out of the data model for now.